camel/pom.xml

```diff
@@ -61,6 +61,10 @@ under the License.
             <artifactId>java-hamcrest</artifactId>
```

Comment:
Love it.  Would we be able to drop the hamcrest dependency after this?

Reply:
Yes, definitely! I'll add a second commit to this PR that drops the hamcrest dependency. (Hamcrest will still be brought in transitively via JUnit, but this way we'd be directly managing fewer dependencies).